nixos/doc/manual/release-notes/rl-1903.xml

```diff
@@ -220,6 +220,13 @@
      reset to the default value (<literal>false</literal>).
     </para>
    </listitem>
+   <listitem>
+    <para>
+     NixOS global firewall allow options (<literal>networking.firewall.allow*</literal>)
+     are now preserved when setting interface specific rules such as
+     <literal>networking.firewall.interfaces.en0.allow*</literal>.
```

Comment:
```suggestion
     <literal>networking.firewall.interfaces.en0.allow*</literal>.
     The pseudo device <literal>networking.firewall.interfaces.default</literal> was removed
     in the process.
```

Reply:
It kind of still uses a pseudo device but named `any` which allows you to overwrite individual rules by assigning to `networking.firewall.interfaces.any.allow*`. Perhaps I should rename that back if it will be less confusing?

Reply:
Yes. That would also increase compatibility.